### PR TITLE
Test some workflows (needs to be merged)

### PR DESCRIPTION
**Pull request by pPris**

### CODE REVIEW

**Comment by pPris** [comment]
@bot ready for review

**Comment by pPris** [comment]
@bot ready for review

File: .github/handle_pr_marked_ready_for_review/main.ts [hunk 1]
```diff
@@ -35,46 +35,53 @@ const owner = github.context.repo.owner;
...
-        } else if (hasLabel(prLabels, "s.ToReview")) {
+        if (hasLabel(prLabels, "s.ToReview")) {
             await dropToReviewLabelAndAddOngoing();
-            await postComment(errMessage);
         }
+
+        // for prs labelled as ongoing, for all other event types except on synchronise, 
+        // we can be sure that the author hasn't been notified of the failing checks by the bot
+        if (hasLabel(prLabels, "s.Ongoing") && isOnSynchronise() && await wasAuthorLinkedToFailingChecks()) {
+            core.info("PR has the ongoing label and author has been notified, exiting...")
```
Comment (pPris) [inline .github/handle_pr_marked_ready_for_review/main.ts:70]: Add single comment

File: .github/handle_pr_marked_ready_for_review/main.ts [hunk 2]
```diff
@@ -35,46 +35,53 @@ const owner = github.context.repo.owner;
...
 
     const prLabels : string[] = await octokit.rest.issues.get({
         owner,
         repo, 
         issue_number
     })
+    .then(res => {core.info(res.headers["x-ratelimit-remaining"]); return res;})
     .then(res => res.data.labels.map(label => label.name || label)) // label may be of type string instead of an object so need this ||
     .then(l => log.info(l, `labels returned for pr ${issue_number}`))
     .catch(err => {core.info(err); throw err});
```
Comment (pizapuzzle) [inline .github/handle_pr_marked_ready_for_review/main.ts:52]: non-author single comment

File: .github/handle_pr_marked_ready_for_review/main.ts [hunk 3]
```diff
@@ -35,46 +35,53 @@ const owner = github.context.repo.owner;
...
-            if (await wasAuthorLinkedToFailingChecks()) {
-                core.info("PR has the ongoing label and author has been warned, exiting...")
-                return;
-            } else {
-                await postComment(errMessage);    
-            }
-        } else if (hasLabel(prLabels, "s.ToReview")) {
+        if (hasLabel(prLabels, "s.ToReview")) {
             await dropToReviewLabelAndAddOngoing();
-            await postComment(errMessage);
```
Comment (pPris) [inline .github/handle_pr_marked_ready_for_review/main.ts:70]: author single comment

**Comment by pPris** [comment]
@bot ready for review

**Comment by pPris** [comment]
@bot ready for review

**Comment by pPris** [comment]
@bot ready for review

**Comment by pPris** [comment]
@bot ready for review

**Pull request merged**